src/compile/render-ssr/handlers/Element.ts

```diff
@@ -105,15 +105,15 @@ export default function(node, renderer, options) {
 				openingTag += '${' + attribute.chunks[0].snippet + ' ? " ' + attribute.name + '" : "" }';
 			} else if (attribute.name === 'class' && classExpr) {
 				addClassAttribute = false;
-				openingTag += ` class="\${ [\`${attribute.stringifyForSsr()}\`, ${classExpr} ].join(' ').trim() }"`;
+				openingTag += ` class="\${[\`${attribute.stringifyForSsr()}\`, ${classExpr}].join(' ').trim() }"`;
```

Comment:
```suggestion
				openingTag += ` class="\${[\`${attribute.stringifyForSsr()}\`, ${classExpr}].join(' ').trim()}"`;
```

?

Reply:
gah, yeah, missed that one. oh well, we'll catch it some other time

Reply:
I mostly wanted to try out the new `` ```suggestion`` stuff, no worries 😄

Reply:
yeah, it's neat, though TIL the button only appears for unmerged PRs